src/styles/utils/media.scss

```
@@ -0,0 +1,17 @@
+@mixin tablet() {
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@mixin laptop() {
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@mixin dasktop() {
```

```suggestion
@mixin desktop() {
```
typo

I'd recommend using [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) extension